client/test/integration/index.ts

```diff
@@ -39,8 +39,8 @@ export async function run(): Promise<void> {
 
     mocha.options.color = true;
     mocha.rootHooks({
-        beforeEach: () => {
-            globalThis.integrationTestHelper.setupEachTest();
+        beforeEach: async () => {
```

Comment:
This raises the question of - why is async/await needed here but not for afterEach? And why is it being included - what problem is being fixed? I think a comment in the code is the right way to describe why.

Reply:
Good point. The teardown is asynchronous also, so I updated that code to use async/await.